<?xml version='1.0' encoding='utf-8'?>

<launch>
    <!--打开物联网设备-->
    <node pkg = "iot_wifi_system" type = "IOT_WIFI_Node" name = "IOT_WIFI_Node" output="screen"/>
    <!--打开语音唤醒-->
    <include file="$(find castlex_voice_system)/launch/castlex_awake.launch"/>
    <!--打开语音合成-->
    <node pkg = "castlex_voice_system" type = "castle_tts_node" name = "castle_tts_node" output="screen">
        <param name="speech_param" value="voice_name = vixy, text_encoding = utf8, sample_rate = 16000, speed = 50, volume = 50, pitch = 50, rdn = 0" />
    </node>
    <!--打开命令词识别-->
    <node pkg = "castlex_voice_system" type = "iot_order_node" name = "iot_order_node" output="screen"/>
    	
</launch>


